### DISCUSSION TITLE
Copy missing data to another server

### DISCUSSION THREAD

**Discussion by khorton** [teslamate-org/teslamate#3233] [Q&A]
I have two Teslamate instances running on separate servers.  One server had a failure and was not running for several hours, so it is missing numerous drives.  I wish to export a few hours of data from the server that was running and import it to the server that had the failure.

How could I export a selected time slice of data and then import it to another Teslamate instance?  

Thanks - Kevin

**Comment by burratha** [comment]
Am also interesting in performing something similar.

**Comment by ktsenvolt** [comment]
Just have to restore the backup data from a good server.

**Reply by khorton** [reply]
But, the server that has the data for the missing drives is missing manually entered charge costs for dozens of charges. How would I do a backup and restore of just a selected time slice?
